src/Maestro/Maestro.Web/Api/v2018_07_16/Controllers/ChannelsController.cs

```diff
@@ -294,5 +307,91 @@ public async Task<IActionResult> DeletePipelineFromChannel(int channelId, int pi
+        /// <param name="includeDisabledSubscriptions">Include disabled subscriptions</param>
+        /// <param name="includedFrequencies">Frequencies to include</param>
+        /// <param name="includeBuildTimes"></param>
+        /// <param name="days">Number of days over which to summarize build times</param>
+        [HttpGet("{channelId}/graph")]
+        [SwaggerApiResponse(HttpStatusCode.OK, Type = typeof(FlowGraph), Description = "The dependency flow graph for a channel")]
+        [ValidateModelState]
+        public async Task<IActionResult> GetFlowGraphAsync(
+            int channelId = 0, 
+            bool includeDisabledSubscriptions = false,
+#pragma warning disable API0001 // Versioned API methods should not expose non-versioned types.
+            IEnumerable<string> includedFrequencies = default,
+#pragma warning restore API0001 // Versioned API methods should not expose non-versioned types.
+            bool includeBuildTimes = false,
+            int days = 7)
+        {
+            var barOnlyRemote = await _remoteFactory.GetBarOnlyRemoteAsync(Logger);
+
+            Microsoft.DotNet.Maestro.Client.Models.Channel engLatestChannel = await barOnlyRemote.GetChannelAsync(engLatestChannelId);
+            Microsoft.DotNet.Maestro.Client.Models.Channel eng3Channel = await barOnlyRemote.GetChannelAsync(eng3ChannelId);
+
+            List<Microsoft.DotNet.Maestro.Client.Models.DefaultChannel> defaultChannels = (await barOnlyRemote.GetDefaultChannelsAsync()).ToList();
+
+            if (engLatestChannel != null)
+            {
+                defaultChannels.Add(
+                    new Microsoft.DotNet.Maestro.Client.Models.DefaultChannel(0, "https://github.com/dotnet/arcade", true)
+                    {
+                        Branch = "master",
+                        Channel = engLatestChannel
+                    }
+                );
+            }
+            if (eng3Channel != null)
+            {
+                defaultChannels.Add(
+                    new Microsoft.DotNet.Maestro.Client.Models.DefaultChannel(0, "https://github.com/dotnet/arcade", true)
+                    {
+                        Branch = "release/3.x",
+                        Channel = eng3Channel
+                    }
+                );
+            }
+
+            List<Microsoft.DotNet.Maestro.Client.Models.Subscription> subscriptions = (await barOnlyRemote.GetSubscriptionsAsync()).ToList();
+
+            // Build, then prune out what we don't want to see if the user specified
+            // channels.
+            DependencyFlowGraph flowGraph = await DependencyFlowGraph.BuildAsync(defaultChannels, subscriptions, barOnlyRemote, days);
+
+           IEnumerable<string> frequencies = includedFrequencies == default || includedFrequencies.Count() == 0 ? 
+                                   new string[] { "everyWeek", "twiceDaily", "everyDay", "everyBuild", "none", } : 
```

Comment:
I'm not... sure what the indentation level is here, but it looks too far.  It should be indented just 4 spaces form the line before it.